### PR TITLE
[INLONG-266] add dataproxy sdk example doc

### LINKED ISSUE #266: [Feature]add dataproxy sdk usage doc
add dataproxy sdk usage doc

### PR DESCRIPTION
**Pull request by baomingyu**
Fixes #266

add dataproxy sdk usage doc

### CODE REVIEW

File: docs/sdk/dataproxy-sdk/usage.md [hunk 1]
```diff
@@ -1,13 +1,197 @@
 ---
 title: Usage
 ---
-# how to use
+# How To Use
+
+## 一、Add dependency in maven pom.xml
```
Comment (healchow) [inline docs/sdk/dataproxy-sdk/usage.md:6]: Please use an English serial number.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md [hunk 2]
```diff
@@ -3,11 +3,186 @@ title: 使用
...
    </dependency>
-```
\ No newline at end of file
+```
+## 二、Example 演示
+  Api详情，请查看[总览](./overview)。<br/>
+  inlong-dataproxy-sdk 中提供了tcp，http两种协议的API，
+  暂未提供UDP方式的API，如果需要采用UDP方式发送数据，需要用户自己按照sdk中的方式进行组包。
+  inlong-dataproxy-sdk 的example 目录提供了tcp、http、udp三种方式的example示例代码，有需要的用户可以参考相应的实现。
+### 2.1 TCP client
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md:19]: client -> Example

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md [hunk 3]
```diff
@@ -3,11 +3,186 @@ title: 使用
...
+| inLongManagerPort      | String   | inlong管理台端口                             |
+| netTag                 | String   | 网络标签，暂未使用，可以传空字符串                |
+| dataProxyGroup         | String   | dataProup组名称，用户在启用本地配置的时候，用于本地配置的名称         |
+| isLocalVisit           | boolean  | 是否使用本地配置， true 使用https 访问管理台，false使用http请求管理台|
+| isReadProxyIPFromLocal | boolean  | 是否从本地配置文件中获取Dataproxy服务器地址信息，本地自测，不能访问管理台的情况下可以配置为true|
+| configBasePath         | String   | 本地配置文件的路径 默认./inlong，isReadProxyIPFromLocal为true时从这个目录查找配置文件/                                 |
+| msgtype                | int      | 消息类型，取值（3，5，7，8），建议使用7，每种消息类型代表一种传递过程中消息的拼装协议,具体请参照sdk的代码实现|
+
+当isReadProxyIPFromLocal为true的时候, 会从本地配置文件中获取Dataproxy的配置信息。<br/>
+本地文件的路径为：<br>
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md:57]: remove `<br>`

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md [hunk 4]
```diff
@@ -3,11 +3,186 @@ title: 使用
...
+```
+    ${configBasePath}
+```
+
+文件名称为：<br>
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md:62]: ditto

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md [hunk 5]
```diff
@@ -3,11 +3,186 @@ title: 使用
...
+```
+    ${dataProxyGroup}.local
+```
+例如：<br/>
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md:66]: ditto

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md [hunk 6]
```diff
@@ -3,11 +3,186 @@ title: 使用
...
+    dataProxyGroup = inlong_test
+```
+则本地文件的全路径名称为：
+```
+    /data/inlong/inlong_test.local
+```
+
+文件配置内容为(json 格式),其中host为DataProxy 服务器地址，port为对应的端口，这需要至少配置两个（可以配置为相同的两项）：
+```json
+    {"isInterVisit":1,"cluster_id":"1","size":1,"switch":1,"address":[{"host":"127.0.0.1",* "port":"46802"},{"host":"127.0.0.1","port":"46802"}]}
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md:78]: remove `*`

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md [hunk 7]
```diff
@@ -3,11 +3,186 @@ title: 使用
...
+
+| 参数名称                 | 类型     | 说明                                        |
+|------------------------|----------|--------------------------------------------|
+| sender                 | HttpProxySender   | 第一步创建的sender                  |
+| inlongGroupId          | String   | inglongGroupId                             |
+| inlongStreamId         | String   | inlongStreamId                             |
+| messageBody            | String   | 发送的消息内容                                |
+| dt                     | long     | 时间戳                               |
+
+### 2.2 HTTP client
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md:103]: client -> Example

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md [hunk 8]
```diff
@@ -3,11 +3,186 @@ title: 使用
...
+参数说明如下：
+
+| 参数名称                 | 类型     | 说明                                        |
+|------------------------|----------|--------------------------------------------|
+| sender                 | HttpProxySender   | 第一步创建的sender                  |
+| inlongGroupId          | String   | inglongGroupId                             |
+| inlongStreamId         | String   | inlongStreamId                             |
+| messageBody            | String   | 发送的消息内容                                |
+
+### 2.3 UDP client
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md:186]: client -> Example

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md [hunk 9]
```diff
@@ -3,11 +3,186 @@ title: 使用
...
+    dataProxyGroup = inlong_test
+```
+
+则本地文件的全路径名称为：
+```
+    /data/inlong/inlong_test.local
+```
+文件配置内容为(json 格式),其中host为DataProxy 服务器地址，port为对应的端口，这需要至少配置两个（可以配置为相同的两项）：
+```json
+    {"isInterVisit":1,"cluster_id":"1","size":1,"switch":1,"address":[{"host":"127.0.0.1",* "port":"46802"},{"host":"127.0.0.1","port":"46802"}]}
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/sdk/dataproxy-sdk/usage.md:163]: remove `*`

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**